<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalBoard</class>
 <widget class="QWidget" name="FinalBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Final Board</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</iconset>
  </property>
  <widget class="QLabel" name="tmLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: aqua
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Total Match Played</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="p1Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>player_1 won</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="p2Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Player_2 won</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="p2Score">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Score Card</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="p1Score">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Score Card</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tmScore">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: aqua
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Score Card</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mdLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: yellow
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>match drawn</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>quit from game</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: red;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/quit.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/quit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>go to main-menu</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: black;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/home.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>play game again with continues score</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: red;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/replay.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/replay.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>100</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>playButton</tabstop>
  <tabstop>homeButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
